src/plugins/csv/csvplugin.cpp

```diff
@@ -43,62 +45,64 @@ CsvPlugin::CsvPlugin()
 
 bool CsvPlugin::write(const Map *map, const QString &fileName)
 {
+    // Extract file name without extension and path
+    QFileInfo fileInfo(fileName);
+    fileInfo.setCaching(false);
```

Comment:
Alright. Actually, I think neither of the operations you're doing will cause it to access the file system anyway, since they just operate on the path.